mappings/net/minecraft/nbt/TagString.mapping

```diff
@@ -9,6 +9,12 @@ CLASS ib net/minecraft/nbt/TagString
 		ARG 2 tracker
 	METHOD a write (Ljava/io/DataOutput;)V
 		ARG 0 output
+	METHOD a prettyPrint (Ljava/lang/String;I)Lix;
```

Comment:
Is prettyPrint a good conventional name to use?

Reply:
It's not. I'd like to propose toComponent or toTextComponent, with keeping of the latest discussins in mind.

Reply:
toTextComponent felt kinda weird at the time but prettyPrint does too.